### PR TITLE
Updating Dashbook and adding an example with info

### PR DESCRIPTION
**Pull request by erickzanardo**
# Description

Updating Dashbook, so we can use the new features from it on examples, and also proposing a format for adding instructions on some examples

https://user-images.githubusercontent.com/835641/116759489-ce163200-a9e8-11eb-9162-b32ac6fa836e.mov

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
- [ ] I updated/added relevant documentation (doc comments with `///`) and updated/added examples in `doc/examples`.
- [x] I have formatted my code with `./scripts/format.sh` and the Flame analyzer (`./scripts/analyze.sh`) does not report any problems.
- [x] I read and followed the [Flame Style Guide].
- [ ] I have added a description of the change under `[next]` in `CHANGELOG.md`.
- [x] I removed the `Draft` status, by clicking on the `Ready for review` button in this PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flame users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in `CHANGELOG.md`).
- [x] No, this is *not* a breaking change.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database]. Indicate, which of these issues are resolved or fixed by this PR.*

[issue database]: https://github.com/flame-engine/flame/issues
[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Flame Style Guide]: https://github.com/flame-engine/flame/blob/main/STYLEGUIDE.md

### CODE REVIEW

**Review by spydon — APPROVED** [review]
Nice, we should do this for all examples eventually! :)

**Comment by erickzanardo** [comment]
> Nice, we should do this for all examples eventually! :) 

Yes! We should organize a task force for that eventually hahahhaha

File: examples/lib/stories/animations/animations.dart [hunk 1]
```diff
@@ -6,12 +6,43 @@ import 'animation_group.dart';
 import 'aseprite.dart';
 import 'basic.dart';
 
+const basicInfo = '''
+Basic example of `SpriteAnimation`s use in Flame's `BaseGame`
+
+The snippet shows how an animation can be loaded and added to the game
+```
```
Comment (luanpotter) [inline examples/lib/stories/animations/animations.dart:13]: does it support ```dart ?

**Review by luanpotter — APPROVED** [review]
AWESOME!

File: examples/lib/stories/animations/animations.dart (hunk 1 above)
Comment (erickzanardo) [reply examples/lib/stories/animations/animations.dart:13]: Unfortunately not :(

**Pull request merged**